back/src/messages/input-processor.ts

```diff
@@ -79,12 +79,10 @@ const handleEstablishConnectionStudent = async (socketInfo: SocketInfo, room: st
     return [];
   }
```

Comment:
Check pods structure in planning poker: https://github.com/Lemoncode/planning-poker-example/tree/master/back/src/pods

Reply:
The pod structure will be implemented in the next iteration, it was not planned to be in the current RP. The current PR is just to check if everything is still working fine with the database.

Do you want me to refactor the app in that way now or in a next PR?

Reply:
Nice, it's ok move it to next iteration. I just want to comment it